### ISSUE TITLE
version upgrade bug

### ISSUE THREAD

**Issue by derrickburns** [jenkins-x/jx#4168]
### Summary 

```
$ jx version
NAME               VERSION
jx                 2.0.209
jenkins x platform 2.0.330
Kubernetes cluster v1.12.6-eks-d69f1b
kubectl            v1.14.1
helm client        Client: v2.14.0+g05811b8
git                git version 2.20.1
Operating System   Mac OS X 10.14.5 build 18F132
WARNING:
A new jx version is available: 2.0.212
? Would you like to upgrade to the new jx version? Yes

You are already on the latest version of jx 2.0.209
```

### Jx version

The output of `jx version` is:

```
COPY OUTPUT HERE
```

### Jenkins type

- [ ] Next Generation (Tekton + Prow)
- [ ] Classic Jenkins
- [ ] Serverless Jenkins (JenkinsFileRunner + Prow)

**Comment by warrenbailey** [comment]
Hi @derrickburns, could do with a few more details on this one. Please fill out the following sections: summary, steps to reproduce, expected behaviour, actual behaviour, kubernetes cluster and jenkins type.

/priority awaiting-more-evidence
/kind bug

**Comment by Wizofgoz** [comment]
I'm having the same problem on jx version 2.0.215
```
Wizofgoz:home mac$ jx version
NAME               VERSION
jx                 2.0.215
Kubernetes cluster v1.11.8-eks-7c34c0
kubectl            v1.14.0
helm client        v2.9.1+g20adb27
helm server        v2.9.1+g20adb27
git                git version 2.16.3
Operating System   Mac OS X 10.14.4 build 18E226
WARNING: 
A new jx version is available: 2.0.268
? Would you like to upgrade to the new jx version? Yes

You are already on the latest version of jx 2.0.215
```

Summary:
Attempts at upgrading cli fail claiming that you are running latest version of jx.

Steps to reproduce:
1. run `jx version`
2. confirm upgrade

Expected behavior:
Successfully upgrade to latest release

Actual behavior:
Upgrade fails and claims to already be running the latest release

Kubernetes cluster:
EKS

Jenkins Type:
Next Generation

**Comment by hferentschik** [comment]
@Wizofgoz Could you re-run the command using the `--verbose` option to get some more trace.
Also, are you using homebrew and did you install jx via `brew`?

**Comment by garethjevans** [comment]
This should be resolved in 2.0.277
